data/tabular/skin_reaction/meta.yaml

```diff
@@ -18,6 +18,8 @@ targets:
           - verb: causes skin reaction
           - verb: causes skin sensitization
           - verb: causes drug induced skin immune reaction
+          - adjective: reactive in contact with the skin
```

Comment:
"reactive in contact" sounds not too common for me?
Maybe switch below to the nouns more often?